conf.py

```diff
@@ -56,6 +56,7 @@
 exclude_patterns = [
     'locale',
     'README.*',
+    '**/include',
```

Comment:
I'm really not a fan of this hack. It looks like you factor content out of `views.rst` only for aesthetics, and it backfires in multiple ways: content built twice, duplicated refs, headings defined in the included file...

Instead, you could insert the RST directly where it's included, and it would be perfectly fine.

Alternatively, you could also create one page per view type and neatly nest them under `views`.

Reply:
It seems to me that there is an option to not build the includes but just the pages that use them. I was counting on this to simplify centralizing the doc makes it possible to update all the places at the same time and not to have an outdated piece. Code of doc aesthetics = easier to maintain. According to your comment I was wrong in the scope of the rst options.

The include is indeed used to not have a gigantic atrocious page to work in. I could split the pages My goal was to have the h1 for the views and h2 for each of the views. Either way, it doesn't change much. This will add titles to the left reference menu.

Reply:
Then maybe go with "Alternatively, you could also create one page per view type and neatly nest them under views." for a start since it requires less work, and we'll see how it looks? FYI, there are UI improvements coming in to better render sub-pages :)

If the proposed structure poses a problem, I think it'd be better to stick with the giant RST file then.